code/game/dna/mutations/mutation_powers.dm

```diff
@@ -900,7 +900,7 @@
 
 
 	//Body accessory.
-	if((M.dna.species.tail && M.dna.species.bodyflags & (HAS_TAIL)) || (M.dna.species.wing && M.dna.species.bodyflags & (HAS_WING)))
+	if((M.dna.species.tail && M.dna.species.bodyflags & (HAS_TAIL)) || (M.dna.species.wing && M.dna.species.bodyflags & (HAS_WING)) || (M.dna.species.spines && M.dna.species.bodyflags & (HAS_BACK_SPINES)))
```

Comment:
is there any reason spines are not just a wing subtype? They seem to consist of a lot of the same visual elements